draft-liu-teas-transport-network-slice-yang.txt

```diff
@@ -82,93 +82,160 @@ Copyright Notice
+   desired service-level objectives (SLOs) and service-level
+   expectations (SLEs) over various constructs within the customized
+   topology.
 
    The defined data model is an interface between customers and
    providers for configurations and state retrievals, so as to support
    network slicing as a service.  Through this model, a customer can
-   learn the slicing capabilities and the available resources of the
-   provider.  A customer can request or negotiate with a network slicing
-   provider to create an instance.  The customer can incrementally
-   update its requirements on individual topology elements in the slice
-   instance, e.g., adding or removing a node or link, updating desired
-   bandwidth of a link, and retrieve the operational states of these
-   elements.  With the help of other mechanisms and data models defined
-   in IETF, the telemetry information can be published to the customer.
-
-   The YANG model defines technology-agnostic constructs common to
-   network slicing at network layers of different technologies, e.g.
-   IP/MPLS(-TP), OTN and WDM.  Therefore, this model may be used as a
-   common base model on which other network slicing models, such as
-   [I-D.ietf-ccamp-yang-otn-slicing], may augments with technology-
-   specific constructs.
+   request or negotiate with a network slicing provider to create an
+   instance.
+   The customer can incrementally update its requirements on individual
+   topology elements in the slice instance, e.g., adding or removing a
+   node or link, updating desired bandwidth of a link, and retrieve the
+   operational states of these elements.  With the help of other
+   mechanisms and data models defined in IETF, the telemetry information
+   can be published to the customer, too.
+
+
+
+
+
+
+
+
+
+Liu, et al.             Expires 31 December 2023                [Page 3]
+
+Internet-Draft      Network Slice Topology Data Model          June 2023
+
+
+   The YANG model defines constructs that are technology-agnostic to
+   network slicing built on network layers with different technologies
+   such as IP/MPLS, MPLS-TP, OTN and WDM optical.  Therefore, this model
+   can serve as a common and base model on which technology-specific
+   models for network slicing, such as
+   [I-D.ietf-ccamp-yang-otn-slicing], may be built.
 
    As described in Section 3 of
    [I-D.contreras-teas-slice-controller-models], the data model defined
-   in this document complements the data model defined in
-   [I-D.ietf-teas-ietf-network-slice-nbi-yang].  In addition to the
-   provider's view, the data model defined in this document models the
-   Type 2 service defined in [RFC8453].
+   in this document is optional for network slicing and complements the
+   data model defined in [I-D.ietf-teas-ietf-network-slice-nbi-yang].
+   In addition to the provider's view, the data model defined in this
+   document models the Type 2 service defined in [RFC8453].
```

Comment:
we need to elaborate a little this point. We need to be careful that if we mention Type 2service i.e. VN type 2, VN is "network provided by a service provider to a customer for the customer to use in any way it wants as though it was a physical network." . Type 2 means with virtual nodes and virtual links . But is probably nothing to do with the customized view we want to describe.

Reply:
The customized view and Type 2 might be related. The nodes and links in the customized topology might be virtual nodes and virtual links.

Reply:
I think the customized topology view is somewhat similar to the single-node virtual node in Type 2 VN, which are both customer's review. 

The confusion for ACTN VN Type 2 is that it is a complex combination of both customized topology (single-node abstraction defined by the customer) and a TE topology (exposed by the provider). In this draft we are kind of separating these two definitions and focus only on the customized topology. 

I think the nodes and links in a customized topology may either share the same ids with what the provider would understand, or reference an abstract topology exposed by the provider.